histomicstk/preprocessing/color_conversion/rgb_to_sda.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+def rgb_to_sda(im_rgb, I_0, allow_negatives=False):
+    """Transform input RGB image or matrix `im_rgb` into SDA (stain
+    darkness) space for color deconvolution.
+
+    Parameters
+    ----------
+    im_rgb : array_like
+        Image (MxNx3) or matrix (3xN) of pixels
+
+    I_0 : float or array_like
+        Background intensity, either per-channel or for all channels
+
+    allow_negatives : bool
+        If True, would-be negative values in the output are clipped to 0
```

Comment:
Did you mean they will be clipped to 0 if this is set to False to not allow negatives?